src/mixins/bonds.js

```diff
@@ -34,7 +34,9 @@ export const BondsMixin = (superclass) => class extends superclass {
     areElementsBonded(element1, coordinate1, element2, coordinate2, bondsData) {
         const distance = Made.math.vDist(coordinate1, coordinate2);
         const connectivityFactor = this.settings.chemicalConnectivityFactor;
-        return Boolean(bondsData.find(b => b.length.value && (b.length.value * connectivityFactor) >= distance));
+        return Boolean(filterBondsDataByElementsAndOrder(bondsData, element1, element2).find(b => {
```

Comment:
Let's clean up the docstrings in this file too, pls.

Reply:
Done.